src/metrics.js

```diff
@@ -78,5 +85,27 @@ module.exports = {
   featureFulfilled,
   getOsName,
   version: kitePkg.version,
-  track: () => {}
+  track: (event, props) => {
+    if (process.env.NODE_ENV === "development") {
```

Comment:
Most likely, one of the flows uses `--mode none`. Why don't we check that the ENV isn't production instead here?